modules/caddyhttp/templates/templates.go

```diff
@@ -246,14 +261,12 @@ func init() {
 //
 // **JSON** is simply `{` and `}`:
 //
-// ```
-//
-//	{
-//		"template": "blog",
-//		"title": "Blog Homepage",
-//		"sitename": "A Caddy site"
-//	}
-//
+// ```json
+// {
+// "template": "blog",
+// "title": "Blog Homepage",
+// "sitename": "A Caddy site"
+// }
 // ```
```

Comment:
This is really stupid, but if we have any kind of indentation between `{ }` the lint-on-save in VSCode will reformat it as how it was before, making it look stupid after markdown parsing. So for now the only solution I could find is to remove any JSON indentation :sob:

Reply:
Oh! Haha, yeah you're right. I don't know a workaround. That's OK.